ParallaxView.podspec

```diff
@@ -6,7 +6,7 @@ Pod::Spec.new do |s|
     s.summary = "Add parallax effect like in tvOS applications to any view."
     s.requires_arc = true
     s.license = "MIT"
-    s.version = "2.0.8"
+    s.version = "5.0"
```

Comment:
Just leave `s.version` as it is and add `s.swift_version = '5.0'`.

Reply:
```suggestion
    s.version = "2.0.8"
```

Reply:
What makes you think that, sir?

Reply:
That's what @RomanPodymov asked for above so this PR can be merged?

>Just leave s.version as it is